src/html/shared/header.html

```diff
@@ -0,0 +1,21 @@
+<nav class="c-header  is-scroll" style="background-color:white;">
```

Comment:
@xavierang let's make this a `<header>` element and move the nav into a nested element